WcaOnRails/app/webpacker/components/Media/MediaTable.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect, useState } from 'react';
+
+import _ from 'lodash';
+import { Table, Confirm,Modal,Button } from 'semantic-ui-react';
+
+import { competitionUrl } from '../../lib/requests/routes.js.erb';
+import { countries } from '../../lib/wca-data.js.erb';
+import { dateRangeBetween } from '../../lib/helpers/media-table';
+import useLoadedData from '../../lib/hooks/useLoadedData';
+import useSaveAction from '../../lib/hooks/useSaveAction';
+
+export default function MediaTable({ isValidate }) {
+  const handleUpdateSuccess = () => {
+    setModalOpen(true);
```

Comment:
Does this line of code work? Trying to access a variable that doesn't get defined until line 28?